scripts/box/connect_external_accounts.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+First run 
+    rm -rf website/addons/box/views/
+
+Then change the user_settings field of BoxNodeSettings to foreign_user_settings
```

Comment:
The docstring should explain the purpose and any pre-migration steps that need to be taken before the script is run. Take out anything that isn't necessary.